Makefile

```diff
@@ -3,13 +3,30 @@
 ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
 $(eval $(ARGS):;@:)
 
+# Dependencies used in development, for ood-gen in particular
```

Comment:
Would it not be easier to track this in an opam file perhaps and `pin -yn` and `install --deps-only`?

Reply:
`opam install --deps-only .` still works on the repo, but installs only what you need to run the server, which made sense for me (if you don't work on the data, you don't need to install anything else)
In development, you can set up your env with `make switch` which will install the dev dependencies for you.
I've copied this workflow from Dune's repo, but I'm not sure what's best.